config/networking-ns-cert.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: networking-ns-cert
+  namespace: knative-serving
+  labels:
+    serving.knative.dev/release: devel
```

Comment:
We need to label this in a manner that it can be excluded from installs similar to our other optional components.

cc @ZhiminXiang

Reply:
How about using the label `networking.knative.dev/tls: namespace` ?

Reply:
Sorry, I missed this one. `networking.knative.dev/tls` seems broad to me. I don't have a strong opinion on this but there seems to be a pattern that `networking.knative.dev/wildcard-certificate-provider: nscert` would follow.

Reply:
That SGTM!